.circleci/config.yml

```diff
@@ -115,6 +116,29 @@ jobs:
       - run: make check
       - run: make checkscripts
 
+  build-binaries:
+    executor: go
+    environment:
+      <<: *COMMON_ENVS
+      GOPATH: /go
+      GO_TAGS: "ui codegen_generated"
+    steps:
+      - checkout
+      - run: apt-get update; apt-get install -y sudo unzip zip
+      - run: make deps
+      - install-protoc
+      - run: sudo -E PATH="$GOPATH/bin:/usr/local/go/bin:$PATH" make generate-structs
+      - run: make pkg/linux_amd64/nomad
+      - run: make pkg/windows_amd64/nomad
+      - run: zip -j pkg/nomad_windows_amd64.zip pkg/windows_amd64/nomad.exe
+      - run: tar -czf pkg/nomad_linux_amd64.tar.gz -C ./pkg/linux_amd64 .
```

Comment:
Should this be a zip file too?  Also, you can do `make pkg/windows_amd64.zip pkg/linux_amd64.zip`

Reply:
Sure, that works.